src/VisualStudio/Core/Def/Implementation/Diagnostics/DiagnosticProgressReporter.cs

```diff
@@ -33,12 +43,15 @@ internal sealed class TaskCenterSolutionAnalysisProgressReporter
             VisualStudioWorkspace workspace)
         {
             _lastTimeReported = DateTimeOffset.UtcNow;
+            _resettableDelay = null;
```

Comment:
this doesn't seem necessary.  how woudl this be non-null in the constructor?

Reply:
I set null even though field will be set to null by default. not first time this is done.